swarm.cabal

```diff
@@ -61,6 +60,7 @@ data-files:
   test/standalone-topography/*.png
   test/standalone-topography/*.yaml
   worlds/*.world
+  example/*.sw
```

Comment:
Ah, I guess we have it in `extra-source-files` because it is not in data directory?

Maybe we could workaround it by a symlink?